python_modules/libraries/dagster-dg/dagster_dg/config.py

```diff
@@ -153,7 +153,10 @@ class DgCliConfig:
         cache_dir (Optional[str]): The directory to use for caching. If None, the default cache will
             be used.
         verbose (bool): If True, log debug information.
-        builitin_component_lib (str): The name of the builtin component library to load.
+        use_component_modules (list[str]): Specify a list of modules containing components.
```

Comment:
sounds like a bool option

Reply:
Why? This lets you say "I want to load components from just these modules". More flexible and more clear than a bool option.

Reply:
To be more specific. A `use_` prefix almost certainly implies to me that it is a bool (as `use_dg_managed_environment` does which is immediately below) so I think we should have a different name